cmd/skywire-cli/commands/visor/gen-config.go

```diff
@@ -26,17 +26,19 @@ var (
 	replace       bool
 	testEnv       bool
 	packageConfig bool
+	skybianConfig bool
 	hypervisor    bool
 	hypervisorPKs string
 )
 
 func init() {
-	genConfigCmd.Flags().Var(&sk, "sk", "if unspecified, a random key pair will be generated.")
+	genConfigCmd.Flags().Var(&sk, "sk", "if unspecified, a random key pair will be generated.\n")
```

Comment:
Why are we adding a newline here specifically?

Reply:
the help menu will print right after that the default value. Without the newline this would wrap around to where the flags are specified. The newline puts it just underneath the description text for the flag, which increases readability.

There are several other help menus places that need similar attention to improve readability that i will submit going forward.